api/yelp_beans/templates/match_email.html

```diff
@@ -19,10 +19,10 @@
                         <img style="border-radius:50%;width:80px" src={{participant.photo_url}}>
                     </td>
                     <td width="30px">
-                        <p><a href="{{participant.metadata.get('company_profile_url')}}">{{participant.first_name}} {{participant.last_name}}</a>
+                        <p><a href="{{participant.metadata['company_profile_url']}}">{{participant.first_name}} {{participant.last_name}}</a>
                         </p>
-                        <p>{{participant.metadata.get('department')}}</p>
-                        <p>{{participant.metadata.get('business_title')}}</p>
+                        <p>{{participant.metadata['department']}}</p>
```

Comment:
Glad you made this change. I realized I'd fixed this in the Twitch repo but not on here. This needs to be meta_data and not metadata